discovery-provider/scripts/start.sh

```diff
@@ -100,7 +100,6 @@ else
         celery -A src.worker.celery beat --loglevel $audius_discprov_loglevel 2>&1 | tee >(logger -t beat) &
     fi
 
-    docker run -d --name watchtower -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower --interval 10
```

Comment:
not sure what this was doing before

Reply:
@cheran-senthil said it's old and should be removed